src/components/dictionaryConcepts/containers/EditConcept.jsx

```diff
@@ -87,6 +86,8 @@ export class EditConcept extends Component {
       openGeneralModal: false,
       url: '',
       mapp: '',
+      isChange: false,
```

Comment:
Do you still need these flags?

Reply:
I guess there is no need, I have refactored and removed them.